### PR TITLE
Adjust Test + Vaccination Certificate Detail Screen (EXPOSUREAPP-7671, 7656)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Updated Details Screens for Details Screens for Vaccination + Test Certificates according to figma
-> Attributes in the Vaccination Detail View will be updated once the VaccinationRepository is updated.

Vaccination Detail - Incomplete:
image

Vaccination Detail - Complete:
image

**Please test:**
Check test certificate by scanning a valid test certificate
Check the vaccination certificate by scanning a valid certificate
 -> if the vaccination is incomplete, the card should be grey
 -> if the vaccination is complete, the card should be blue

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/details/CovidCertificateDetailsFragment.kt [hunk 1]
```diff
@@ -63,13 +64,8 @@ class CovidCertificateDetailsFragment : Fragment(R.layout.fragment_covid_certifi
...
         testCertificate: TestCertificate
     ) {
-        qrCodeCard.subtitle.text = getString(
-            R.string.detail_green_certificate_card_subtitle,
-            testCertificate.sampleCollectedAt.toShortDayFormat(),
-            testCertificate.sampleCollectedAt.toShortTimeFormat(),
-        )
         name.text = testCertificate.run { "$lastName, $firstName" }
-        birthDate.text = testCertificate.dateOfBirth.toDayFormat()
+        birthDate.text = testCertificate.dateOfBirth.toString(dayOfBirthFormatter)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/details/CovidCertificateDetailsFragment.kt:68]: is it a birthdate specific format ? can we create an extension function for that if it has to be for all birthdates?
`toBirthdateFormat()`

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/ui/details/VaccinationDetailsFragment.kt [hunk 2]
```diff
@@ -84,37 +98,51 @@ class VaccinationDetailsFragment : Fragment(R.layout.fragment_vaccination_detail
...
     private fun FragmentVaccinationDetailsBinding.bindCertificateViews(
         certificate: VaccinationCertificate
     ) {
         name.text = certificate.fullName
-        birthDate.text = getString(
-            R.string.vaccination_details_birth_date,
-            certificate.dateOfBirth.toDayFormat()
-        )
-        vaccinatedAt.text = certificate.vaccinatedAt.toDayFormat()
+        dateOfBirth.text = certificate.dateOfBirth.toString(dayOfBirthFormatter)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/ui/details/VaccinationDetailsFragment.kt:118]: I thin extension function for formatting birthdate would be great

**Review by mtwalli — DISMISSED** [review]
Lgtm

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/ui/details/VaccinationDetailsFragment.kt (hunk 2 above)
Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/ui/details/VaccinationDetailsFragment.kt:118]: Yes, good point! I will add it

**Review by mtwalli — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by SamuraiKek — APPROVED** [review]
LGTM

**Pull request merged**